sklearn/ensemble/_hist_gradient_boosting/tests/test_splitting.py

```diff
@@ -43,11 +43,14 @@ def test_histogram_split(n_bins):
                                           dtype=np.uint32)
             has_missing_values = np.array([False] * X_binned.shape[1],
                                           dtype=np.uint8)
+            monotonic_cst = np.array([False] * X_binned.shape[1],
```

Comment:
should this not be `0`? It's not really a boolean logic here.

Reply:
Yes, this should actually be `NO_CST`.